index.js

```diff
@@ -1,14 +1,37 @@
+const {
+  fetchJSON,
+  getOwner,
+  getRepo
+} = require('./util')
 /**
  * This is the entry point for your Probot App.
  * @param {import('probot').Application} app - Probot's Application class.
  */
+var base64 = require('base-64')
 module.exports = app => {
   // Your code here
   app.log('Yay, the app was loaded!')
 
   app.on('issues.opened', async context => {
-    context.log(typeof (context.payload.issue.body))
+    const issueBody = (context.payload.issue.body)
     const issueComment = context.issue({ body: 'Thanks for opening this issue!' })
+    const owner = getOwner(context)
+    const repo = getRepo(context)
+    /**
+    * Fetch contents of readme.md containing
+    * content in base64 format
+    */
+    const templateBody = await fetchJSON(
+      `https://api.github.com/repos/${owner}/${repo}/contents/README.md`
```

Comment:
Since we're beginning with fetching the contents of the template as our first step, I think it's best to point out at this stage itself that there are [two possible ways](https://blog.github.com/2016-02-17-issue-and-pull-request-templates/) to add Issue/Pull Request Template to a repo. I would suggest hitting both, with the `.github` one being first preference followed by the root directory one. It's a subtle point best addressed in this PR itself I feel.